styleguide/screens/Patterns/Tether/Tether.js

```diff
@@ -0,0 +1,182 @@
+    const { value: attachment } = e.target;
+
+    this.setState({
+      verticalAttachment: VERTICAL_ATTACHMENTS[attachment],
+    });
+  };
+
+  handleHorizontallAttachmentChange = (e) => {
+    const { value: attachment } = e.target;
+
+    this.setState({
+      horizontalAttachment: HORIZONTAL_ATTACHMENTS[attachment],
+    });
+  };
+
+  handleFlushVerticalChange = (e) => {
+    const { checked: flushVertical } = e.target;
+
+    this.setState({
+      flushVertical,
+    });
+  };
+
+  handleFlushHorizontalChange = (e) => {
+    const { checked: flushHorizontal } = e.target;
+
+    this.setState({
+      flushHorizontal,
+    });
+  };
+
+  render() {
+    const {
+      verticalAttachment,
+      horizontalAttachment,
+      flushVertical,
+      flushHorizontal,
+    } = this.state;
+
+    return (
+      <div>
+        <H level={ 1 }>Tether</H>
+        <T elm="p" className={ cx(m.mtr, m.largeI, m.demi) }>
+          The <C>Tether</C> component allows you define and manage the position of an element
+          in relation to another element.
+        </T>
+        <D>
+          <T elm="p" className={ m.mtr }>
+            Tether is useful where you need to position a element in relation to another element.
+            It’s great as the foundation for other components such as { ' ' }
+            <Link className={ scaffoldCss.link } to="/patterns/dropdown">Dropdown</Link>.
+          </T>
+          <T elm="p" className={ m.mtr }>
+            A useful fact about the Tether component is that it will reposition itself back into
+            the viewport if incorrect horizontal and vertical positioning or scrolling cause it to
+            move outside of the viewport.
+          </T>
+          <Note className={ m.mtr }>
+            <T elm="p">
+              It is best to avoid using the <C>Tether</C> component with a target with { ' ' }
+              <C>position: fixed</C> as it will break the repositioning functionality.
+            </T>
+          </Note>
+          <Specimen
+            classNames={ {
+              root: m.mtr,
+              specimenContainer: cx(m.par, css.specimenContainer),
+            } }
+            code={ dedent`
+              <Tether
+                target={ <button>Anchor</button> }
+                verticalAttachment={ ${verticalAttachment} }
+                horizontalAttachment={ ${horizontalAttachment} }
+                active
+              >
+                <div />
+              </Tether>
+            ` }
+          >
+            <Tether
+              target={ <Target /> }
+              verticalAttachment={ verticalAttachment }
+              horizontalAttachment={ horizontalAttachment }
+              flushVertical={ flushVertical }
+              flushHorizontal={ flushHorizontal }
+              active
+            >
+              <TetherChild className={ css.tether } />
+            </Tether>
+            <div className={ css.controls } >
```

Comment:
I don't think the controls should be inside the specimen's container. Does it not make more sense to have them horizontally across the top of the container? I also think we can use our own components in this case else it'll look a bit naff